edge_computing/image-base-install/ibi-edge-image-based-install.adoc

```diff
@@ -0,0 +1,32 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ibi-image-based-install_{context}"]
+= Performing a deployment for {sno} clusters using zero-touch-configuration
+include::_attributes/common-attributes.adoc[]
+:context: ibi-edge-image-based-install
+
+toc::[]
+
+When a host preinstalled with {sno} arrives at a remote site, a technician can easily deploy the host in a matter of minutes using zero-touch-configuration (ZTC). ZTC uses the Image Based Install (IBI) Operator to configure the host and complete the deployment. 
```

Comment:
Are we still using ZTC? I wasn't sure we were continuing with that name.

Reply:
@romfreiman - would you know if we are still using the phrase "zero touch configuration (ZTC)" for the deployment stage of an image-based install? I have been using it in drafting the IBI docs so far... thanks!